src/python/pants/backend/java/dependency_inference/rules_test.py

```diff
@@ -29,8 +29,8 @@
     Targets,
 )
 from pants.jvm.jdk_rules import rules as java_util_rules
-from pants.jvm.resolve.coursier_fetch import rules as coursier_fetch_rules
 from pants.jvm.resolve.coursier_setup import rules as coursier_setup_rules
+from pants.jvm.resolve.user_resolves import rules as coursier_fetch_rules
```

Comment:
It might be worth doing a scan for `as coursier_fetch_rules` and change that to `as user_resolves_rules`